public/components/layout.js

```diff
@@ -136,19 +160,38 @@ class UserResource extends HTMLElement {
                 return obj
             })
             .catch(err => {
-                alert("Target URI could not be resolved."
-                    + " Certain data previews will not be available."
-                    + " Applications that will use the data you are about to create"
-                    + " will note be able to gather additional information about this targeted resource."
-                    + " You can supply a different URI or continue with this one.")
+                document.getElementById("notResolvedMessage").classList.toggle("show")
                 uriPreview.innerHTML = `<pre>{Not Resolvable}</pre>`
                 localStorage.setItem('userResource', objCreator.value? JSON.stringify({'@id':target, 'creator':objCreator.value}): JSON.stringify({'@id':target}))
                 return null
             })
+        this.validateContext(targetObj)
         //This might help mobile views
         //window.scrollTo(0, confirmURI.offsetTop)
     }
 
+    validateContext(target_object){
+        confirmURI.classList.remove("is-hidden")
+        if(target_object){
+            let providedObj = JSON.parse(localStorage.getItem("userResource"))
+            const context = providedObj["@context"]
+            const acceptableContexts = [
+                "http://iiif.io/api/presentation/3/context.json", 
+                "https://iiif.io/api/presentation/3/context.json"
+            ]
+            // Check if object contains the proper context
+            if(!(context && acceptableContexts.includes(context))){
```

Comment:
Not quite.  This logic fails when I supply the resource https://iiif.io/api/cookbook/recipe/0154-geo-extension/manifest.json.  It has the correct context but still gets the warning.

Reply:
The main thing you missed is that the variable `context` can be an array here.

Reply:
My apologies, I copied the validation code from `navplace.html` and assumed it was correct.

Reply:
@thehabes The resource https://iiif.io/api/cookbook/recipe/0154-geo-extension/manifest.json does not have a context that is in the `acceptableContexts` in navPlace.html (main branch). the below image shows a side-by-side of the contexts from the URI (left) and the acceptable contexts (right). 
So is this a situation where `acceptableContexts` needs to be updated or the URI is not acceptable?
![img](https://github.com/CenterForDigitalHumanities/geolocator/assets/144740070/75c8bbd1-df24-4d1f-8cc0-ee5859f9d73e)

Reply:
https://iiif.io/api/cookbook/recipe/0154-geo-extension/manifest.json has two contexts

- http://iiif.io/api/extension/navplace/context.json
- http://iiif.io/api/presentation/3/context.json

It is required to have one of the following contexts
- https://iiif.io/api/presentation/3/context.json
- http://iiif.io/api/presentation/3/context.json

Therefore, the @context of https://iiif.io/api/cookbook/recipe/0154-geo-extension/manifest.json is valid because that context array contains http://iiif.io/api/presentation/3/context.json

Reply:
> My apologies, I copied the validation code from navplace.html and assumed it was correct.

I can confirm it seems like this code was incorrect.  It is throwing the `alert()` in the same situation.